pyface/tasks/tests/test_task_window.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+
+from traits.testing.unittest_tools import UnittestTools
+from pyface.tasks.api import Task
+from ..task_window import TaskWindow
+
+
+class TestTaskWindow(unittest.TestCase, UnittestTools):
+
+    def test_title(self):
+        task_1 = Task(name='Test Task')
+        task_2 = Task(name='Test Task 2')
+        task_window = TaskWindow(tasks=[task_1, task_2])
+
+        # initially nothing
```

Comment:
I would recommend to break each of these sections to a separate test to make them more granular.